shell/platform/fuchsia/flutter/external_view_embedder.cc

```diff
@@ -115,6 +117,10 @@ void ExternalViewEmbedder::BeginFrame(
 void ExternalViewEmbedder::PrepareFlutterView(int64_t flutter_view_id,
                                               SkISize frame_size,
                                               double device_pixel_ratio) {
+  // TODO(team-fuchsia): Support multiple views. For now, Fuchsia can render
+  // only into the implicit view.
```

Comment:
I think it's better just say we do not support it for now. It's not a TODO if nobody is planning to do it. At least until someone decides to take over this part.
```suggestion
  // Fuchsia only supports operating the implicit view for now.
```